src/components/sections/two-columns.json

```diff
@@ -0,0 +1,75 @@
+{
+  "collectionName": "components_sections_two_columns",
+  "info": {
+    "displayName": "twoColumns",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "name": {
+      "type": "string",
+      "required": true,
+      "maxLength": 256
+    },
+    "isBottomWaved": {
+      "type": "boolean",
+      "default": false,
+      "required": true
+    },
+    "ratio": {
+      "type": "enumeration",
+      "enum": [
+        "ratio 1:1",
+        "ratio 2:1"
+      ],
+      "default": "ratio 1:1"
+    },
+    "backgroundColor": {
+      "type": "enumeration",
+      "enum": [
+        "white",
+        "black",
+        "background",
+        "primary",
+        "secondary",
+        "tertiary"
+      ],
+      "default": "background"
+    },
+    "optionRichText": {
```

Comment:
Question:

I don't understand why these flags are needed. We need to receive columns; the column's content may differ depending on what the Content Manager puts there.